test/e2e/cron_test.go

```diff
@@ -458,25 +456,16 @@ spec:
 
 func (s *CronSuite) TestMalformedCronWorkflow() {
 	s.Given().
-		Exec("kubectl", []string{"apply", "-f", "testdata/malformed/malformed-cronworkflow.yaml"}, fixtures.NoError).
+		Exec("kubectl", []string{"apply", "-f", "testdata/malformed/malformed-cronworkflow.yaml"}, fixtures.ErrorOutput("unknown field \"spec.workflowSpec.arguments.parameters.someParam\"")).
```

Comment:
This test feels odd. Why did it apparently test a wellformed-cronworkflow..., who knows? Just as a delay for the Events to get emitted?

Anyway, can we drop the wellformed workflow from this test, or if it isn't tested elsewhere it should be moved to be in it's own test.

Reply:
I'm not sure either, so I removed it in d3f2ee7377a1bd03ec470d44ac5cb491cfa55fb5